pkgs/development/python-modules/docker/default.nix

```diff
@@ -58,10 +57,7 @@ buildPythonPackage rec {
 
   nativeCheckInputs = [
     pytestCheckHook
-    paramiko
-    pynacl
-    websocket-client
-  ];
+  ] ++ lib.flatten (lib.attrValues optional-dependencies);
```

Comment:
see https://github.com/docker/docker-py/blob/7.1.0/.github/workflows/ci.yml#L50 and we want to follow upstream here